src/AspNetCore/Src/inprocessapplication.cxx

```diff
@@ -651,6 +652,31 @@ IN_PROCESS_APPLICATION::ExecuteApplication(
     return hr;
 }
 
+HRESULT
```

Comment:
No one will use this returned HRESULT, as we are in a callback and cannot interrupt the main thread.